resources/views/partials/bootstrap-table.blade.php

```diff
@@ -537,7 +537,7 @@ function sumFormatter(data) {
 
     $(function () {
         $('#bulkEdit').click(function () {
-            var selectedIds = $('.snipe-table').bootstrapTable('getSelections');
```

Comment:
Wouldn't changing this selector cause issues for the other places where we do bulk editing and we're expecting `.snipe-table` to be the selector?

Reply:
(Bulk model edit, for example)

Reply:
We use the class selector instead of an ID so that we can re-use that bulk method across multiple views, so I don't think changing that behavior is the answer here. 

` class="table table-striped snipe-table"` in that same section *should* be correctly associating the selector.